spec/immutable-data-structures.html

```diff
@@ -21,10 +21,11 @@ <h1>Record ( _arg_ )</h1>
         <emu-alg>
           1. If NewTarget is not *undefined*, throw a *TypeError* exception.
           1. Let _obj_ be ? ToObject(_arg_).
-          1. Let _nameList_ be ? EnumerableOwnPropertyNames(_obj_, ~key~).
+          1. Let _props_ be ? EnumerableOwnPropertyNames(_obj_, ~key+value~).
           1. Let _fields_ be a new empty List.
-          1. For each element _name_ of _nameList_,
-            1. Let _value_ be ? GetV(_obj_, _name_).
+          1. For each element _prop_ of _props_,
+            1. Let _name_ be ? GetV(_obj_, `"0"`).
+            1. Let _value_ be ? GetV(_obj_, `"1"`).

```

Comment:
```suggestion
            1. Let _name_ be ! GetV(_prop_, `"0"`).
            1. Let _value_ be ! GetV(_prop_, `"1"`).
```
(the `!` is since we know these can't fail, because of `EnumerableOwnPropertyNames`'s algorithm)

Reply:
apologies, part of my broken push, I've pushed a commit that contains all of @ljharb 's suggested changes. Should be good now.